src/aten/Pow.cpp

```diff
@@ -0,0 +1,55 @@
+#include <ATen/ScalarOps.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <ATen/native/BinaryOps.h>
+#include <ATen/native/TensorIterator.h>
+
+#include <aten/sycl/PowKernels.h>
+
+namespace at {
+
+Tensor& XPUNativeFunctions::pow_out(
+    const Tensor& base,
+    const Tensor& exp,
+    Tensor& result) {
+  auto iter = TensorIterator::binary_op(result, base, exp);
+  native::xpu::pow_tensor_tensor_kernel(iter);
+  return result;
+}
+
+Tensor& XPUNativeFunctions::pow_out(
+    const Scalar& base,
+    const Tensor& exp,
+    Tensor& result) {
+  if (base.isComplex() && base.toComplexDouble() == 1.0) {
```

Comment:
Please follow the helpers usage.
![image](https://github.com/intel/torch-xpu-ops/assets/28250760/e439546a-d145-4660-b469-6362b3372dde)